server/lib/data_import/my_anime_list.rb

```diff
@@ -0,0 +1,43 @@
+module DataImport
+  class MyAnimeList
+    ATARASHII_API_HOST = "https://hbv3-mal-api.herokuapp.com/2.1/"
+
+    include DataImport::Media
+    include DataImport::HTTP
+
+    attr_reader :opts
+
+    def initialize(opts = {})
+      @opts = opts.with_indifferent_access
+      super()
+    end
+
+    def get_media(external_id) # anime/1234 or manga/1234
+      media = Mapping.lookup('myanimelist', external_id) || external_id.split('/').first.classify.constantize.new # should return Anime.new or Manga.new
```

Comment:
This line is a bit long.  I'd suggest splitting it into three: one to lookup the Mapping, one to identify the class, and one to create the new instance with a `||=`